src/PostgisRegisterTypesPlugin.ts

```diff
@@ -115,7 +187,21 @@ const plugin: Plugin = builder => {
               sql.literal("__gisType"),
               sql.fragment`${sql.identifier(
                 POSTGIS.namespaceName || "public",
-                "geometrytype" // MUST be lowercase!
+                "postgis_type_name" // MUST be lowercase!
+              )}(
+                ${sql.identifier(
+                  POSTGIS.namespaceName || "public",
```

Comment:
The need for this was fixed by https://github.com/graphile/graphile-engine/pull/417; if you declare minimum version requirement as the current postgraphile@next then all should be fine and you can simplify this code.

Not a big deal though.

Reply:
I'll go ahead and simplify.

The current package.json has: 
```
"peerDependencies": {
    "graphile-build": "^4.3.0",
    "graphile-build-pg": "^4.3.0",
    "graphql": ">=0.6 <15",
    "pg-sql2": "^2.2.1"
  }
```

Mind if I drop those and use `build.hasVersion` to check plugin deps?

Reply:
Sure 🤷‍♂️

Reply:
Nevermind, it's a TypeScript thing that I'm not going to mess with in this PR.. deferring this for now.